src/components/Dashboard/Dashboard.css

```diff
@@ -0,0 +1,7 @@
+#dashboard {
+  background: url('../../static/placeholder/1280x720.png');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  height: calc(100vh - 120px);
```

Comment:
@MartyTheeMartian Let's not use `vh`, because it vertically scrunches what the user sees.  They will need to be able to scroll up and down to see, for example, the task hierarchy and task list.